src/epochs.lua

```diff
@@ -346,13 +336,12 @@ end
 
 --- Gets the epoch timestamps for an epoch index
 --- @param epochIndex number The epoch index
---- @return number, number, number # 	The epoch start timestamp, epoch end timestamp, and epoch distribution timestamp
+--- @return number, number # 	The epoch start timestamp, epoch end timestamp, and epoch distribution timestamp
```

Comment:
Comment is out of date (still implies a third return value)

Reply:
All usages seem to be correctly updated for 2 return values.

Reply:
updated comment in https://github.com/ar-io/ar-io-network-process/pull/353/commits/11b9e1c33bb0975520a93ec5193b074670982a3f